index.js

```diff
@@ -51,13 +51,6 @@ export const startEvaluate = async ({
       recordTelemetry,
       logger
     }).catch(err => {
-      // See https://github.com/filecoin-station/spark-evaluate/issues/36
-      // Because each error message contains unique CID, Sentry is not able to group these errors
-      // Let's wrap the error message in a new Error object as a cause
-      if (typeof err === 'string' && err.match(/ENOENT: no such file or directory, open.*\/bafy/)) {
-        err = new Error('web3.storage cannot find block\'s temp file', { cause: err })
-      }
-
```

Comment:
IIRC, we were discussing moving this block from `index.js` down into `preprocess()` function, but maybe I misunderstood your intention.

I don't mind to remove this block entirely if you are sure such change will not degrade the quality of Sentry issue / will not introduce duplicate issues.

Reply:
Ah sorry I mentioned this in the commit message but should have made it more explicit here. This error isn't happening any more (not found in papertrail or sentry), so to me it wasn't worth the complexity of refactoring it

Reply:
That makes sense. I agree that we can remove this block if we are no longer triggering this error. I vaguely remember that we might have eliminated the problem by refactoring our fetch function to call the IPFS GW instead of using the web3 storage client SDK.

Please add the explanation to the commit message, our future selves will thank you.